routes/signup.js

```diff
@@ -8,7 +8,7 @@ module.exports = function(transaction, callback) {
     form: {
       format: 'html',
       lang: transaction.locale,
-      newsletters: 'mozilla-foundation',
+      newsletters: transaction.newsletters || 'mozilla-foundation',
```

Comment:
So the submit line should look like `submit={["email", "country", "newsletters"]}`